tests/unit/printver.tcl

```diff
@@ -1,6 +1,8 @@
 start_server {} {
-    set i [r info]
-    regexp {redis_version:(.*?)\r\n} $i - version
-    regexp {redis_git_sha1:(.*?)\r\n} $i - sha1
-    puts "Testing Redis version $version ($sha1)"
+    test "info command - version and get-sha1" {
+	set i [r info]
+	regexp {redis_version:(.*?)\r\n} $i - version
+	regexp {server_git_sha1:(.*?)\r\n} $i - sha1
+	puts "Testing Redis version $version ($sha1)"
```

Comment:
"Testing Server version...

Reply:
Since we are keeping the `redis_version` info field, and since that is the thing being tested here, I don't think it makes sense to change the output string.